charts/cluster-api-runtime-extensions-nutanix/templates/role.yaml

```diff
@@ -49,17 +42,6 @@ rules:
       - addons.cluster.x-k8s.io
     resources:
       - clusterresourcesets
-    verbs:
```

Comment:
why is this changed?

Reply:
Changes in controller-gen I guess - this is a generated file.